package.json

```diff
@@ -20,13 +20,8 @@
     ]
   },
   "files": [
-    "package.json",
-    "CODE_OF_CONDUCT.md",
-    "CONTRIBUTING.md",
-    "LICENSE",
-    "ios",
     "android",
-    "index.js",
```

Comment:
Could you add these files back, please? In particular, removing the `index.js` would break deploying new versions (since only the files listed here will get pushed to npm)

Reply:
`index.js` is set as `main` in `package.json`. Just like a bunch of other files, it is _always_ included in the packed version, and there no need to manually specify it. You can just do a quick `npm pack --dry-run` locally to see that it is included even though it's not in the files list.